### ISSUE TITLE
 Failed to compile plugin LoF: Can't locate List/MoreUtils.pm in @INC

### ISSUE THREAD

**Issue by jiehuang001** [konradjk/loftee#62]
Hi, I am trying to run loftee on Linux. Below is my command and error message.

Can someone please kindly let me know what is wrong here?

Thank you & best regards,
jie

=== my command ===
vep -i all.tmp6.vcf --vcf --cache --dir /mnt/d/data/vep_cache --offline --assembly GRCh38 -o all.tmp7.vcf --force_overwrite --sift b --canonical --symbol --plugin LoF,loftee_path:/mnt/d/software_lin/loftee,human_ancestor_fa:/d/mnt/data/gatk_bundle/hg38/Homo_sapiens_assembly38.fasta.gz --dir_plugins /mnt/d/software_lin/loftee

=== the error ===
Smartmatch is experimental at /mnt/d/software_lin/loftee/de_novo_donor.pl line 175.
Smartmatch is experimental at /mnt/d/software_lin/loftee/de_novo_donor.pl line 214.
Smartmatch is experimental at /mnt/d/software_lin/loftee/splice_site_scan.pl line 191.
Smartmatch is experimental at /mnt/d/software_lin/loftee/splice_site_scan.pl line 194.
Smartmatch is experimental at /mnt/d/software_lin/loftee/splice_site_scan.pl line 238.
Smartmatch is experimental at /mnt/d/software_lin/loftee/splice_site_scan.pl line 241.
WARNING: Failed to compile plugin LoF: Can't locate List/MoreUtils.pm in @INC (you may need to install the List::MoreUtils module) (@INC contains: /mnt/d/software_lin/loftee /mnt/d/software_lin/ensembl_vep/modules /mnt/d/software_lin/ensembl_vep /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.26.1 /usr/local/share/perl/5.26.1 /usr/lib/x86_64-linux-gnu/perl5/5.26 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.26 /usr/share/perl/5.26 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /mnt/d/software_lin/loftee/svm.pl line 5.
BEGIN failed--compilation aborted at /mnt/d/software_lin/loftee/svm.pl line 5.
Compilation failed in require at /mnt/d/software_lin/loftee/LoF.pm line 34.
Compilation failed in require at (eval 32) line 2.
BEGIN failed--compilation aborted at (eval 32) line 2.

**Comment by Ninjahelicopter** [comment]
Hi,
did you fix your issue?

**Comment by jiehuang001** [comment]
yes, after i followed "Additional components" instruction at https://useast.ensembl.org/info/docs/tools/vep/script/vep_download.html
